### DISCUSSION TITLE
Is there a way to "inspect" the bar to find the right css selectors?

### DISCUSSION THREAD

**Discussion by holty07** [amnweb/yasb#60] [Q&A] [answered]
I'm having trouble getting the traffic widget to show how I want. I can't get it to stay right aligned and a lot of the code I write for the .traffic-widget class doesn't seem to affect it much. I was hoping to have {download_speed} and {upload_speed} with a matching width so there is no movement when the speeds change. For example I like the nice alignment I have in the second image below.
![image](https://github.com/user-attachments/assets/cf14f4b2-8260-4beb-9576-f6eabd092736)
![image](https://github.com/user-attachments/assets/d7de5050-1a78-4c62-bae8-c016f0121662)

**Comment by rcompier** [comment]
I'm having the same issue right now.
Currently I just set the `.traffic-widget` with a min/max width so at least it doesn't jump around.
Still trying to find a way to align it to the left instead of right but no luck so far.
<img width="235" height="57" alt="image" src="https://github.com/user-attachments/assets/11ae5bf3-4629-4346-9644-3bedee12e460" />

**Reply by amnweb** [reply]
Maybe looking into the [documentation](https://github.com/amnweb/yasb/wiki/(Widget)-Traffic) could help you?
Suggestion, set `speed_threshold`,  `max_label_length` and `max_label_length_align`

**Comment by holty07** [comment, marked as answer]
So ChatGPT helped me get what I wanted here. I changed my label to the below to have it exactly as I wanted.

`<table width='96' cellspacing='0' cellpadding='0'>
        <tr><td align='left'>U:</td><td align='right'>{upload_speed}</td></tr>
        <tr><td align='left'>D:</td><td align='right'>{download_speed}</td></tr>
      </table>`
